quality_control/views/qc_inspection_view.xml

```diff
@@ -86,6 +86,8 @@
                                            groups="product.group_uom"
                                            attrs="{'readonly': [('question_type', '=', 'qualitative')]}" />
                                     <field name="test_uom_category" invisible="1"/>
+                                    <field name="min_value" invisible="1"/>
```

Comment:
Why adding this?

Reply:
I noticed that [here](https://github.com/OCA/manufacture/pull/242/files#diff-2235da0e3e8b1a36e273264aa3309e16R243) max_value and min_value were always 0, so I figured out that Odoo couldn't retrieve their value because they were not present in the grid.
That's why I added them in the grid

Reply:
That's not the solution: view is independent from ORM code.

Reply:
Then I'm sorry but I don't know another way to do this.. could you suggest one?
I'll try to explain myself more clearly:
The inspection record has the correct values in min_value and max_value.
During the editing of the inspection, the depends is triggered but [here](https://github.com/OCA/manufacture/pull/242/files#diff-2235da0e3e8b1a36e273264aa3309e16R243) min_value and max_value are 0, I thought that was because their values are fetched from the view (for the little I know, during editing a temporary record is created gathering values from fields in the view).
Then when I save, the calculations [again here](https://github.com/OCA/manufacture/pull/242/files#diff-2235da0e3e8b1a36e273264aa3309e16R243) are made with correct values, fetched from the inspection record.

Reply:
@pedrobaeza in my first answer I didn't mention that the issue is during the edit, is it more clear now why I did this? Is this still the wrong way? 
Thanks for the review

Reply:
@pedrobaeza after the clarifications, is this still the wrong way to do it?
Thanks

Reply:
I see something strange in this, but I don't have time to check the inner problem, so if this solves your problem, let's continue with it.